webapp/app/[locale]/_components/navbar/_components/navItem.tsx

```diff
@@ -128,14 +129,17 @@ const PageLink = function ({
   href,
   icon,
   rightSection,
+  selectedByBaseUrl,
   text,
 }: ItemLinkProps) {
   const locale = useLocale()
   const [networkType] = useNetworkType()
   const pathname = usePathname()
   const { track } = useUmami()
 
-  const selected = pathname.startsWith(`/${locale}${href}`)
+  const selected = selectedByBaseUrl
+    ? pathname.startsWith(`/${locale}/${href.toString().split('/')[1]}/`)
+    : pathname.startsWith(`/${locale}${href}`)
```

Comment:
Instead of sending a boolean flag, I would directly allow overriding the URL to check by props, defaulting to  the current `/${locale}${href}`.

So for example, there would be an optional prop named `compareUrl` (hope you can come up with a better name 😓 ) that defaults to `/${locale}${href}` and then we just

```suggestion
const selected = pathname.startsWith(compareUrl)
```

All the existing components stay the same. For the `Stake` link, you would pass `compareUrl={`/${locale}/stake`}` (You can even make `locale` part of the navItem, so you would just do `compareUrl="/stake"` and then merge with locale in this component.


Thoughts?

Reply:
@dvnahuel when clicking the nav bar, should it go to the stake page or to the dashboard?

Reply:
@gndelia Personally I'm not sure what is the best way, both are similar though I can go with your suggestion then. Why is the Tunnel tab never getting selected? I mean, by clicking on Tunnel, it never gets selected (orange).

Reply:
ah, it should be selected! It seems you found a bug 😄  I will create it

Reply:
https://github.com/hemilabs/ui-monorepo/issues/767

Reply:
Done. Created a new property to override the url that will be used to compare with current pathname.